frontend/packages/enduser-frontend/src/localization/app/en.json

```diff
@@ -639,6 +639,9 @@
           "description": "Finnish and Swedish pre-primary education begins on 11 August 2021.",
           "description2": "<br/>If you need early childhood education starting from 1 August 2021 before the start of pre-primary education, you can apply for it with this application by selecting “I also apply for early childhood education related to pre-primary education”.",
           "label": "Start date",
+          "instructions": {
+            "text": "You can postpone your desired start date until the processing of your application starts. After this, you can make changes to your desired start date by contacting pre-primary education service counselling (tel. 09 816 31000)."
```

Comment:
Do we have any way of knowing when to update this or should we rather link to an external reference (espoo.fi)?

Reply:
There is no exact date for a single application in terms of processing. In any case, most of the applicants will apply either on the first day of the pre-primary education or even earlier.

Reply:
Sry no, I meant the phone number :D

Reply:
I suggest we leave it as it is. Customer seems to be very familiar with the contents of the application and adding extra step by linking to espoo.fi would (imo) make it just more confusing.